src/components/layouts/Table/Table.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Table: React.FC<{ className?: string }> = ({ className = '', children }): JSX.Element => {
```

Comment:
I am not sure if there is another way to do it with the className typing @Frenkiee

As well is that correct to pass classname to potentially override the styling of the component ? 
Wondering if there is a better approach for that or that's fine

Reply:
Well, it worked for me to use this format. My solution is the same as this, but I didn't really dig into it

Reply:
Would also suggest a folder move from src/components/layouts/Table/... to src/components/common/Table/...